File: js10ejercicios/6.Formas_Geometricas/Control.js
const Circulo = require("./Circulo");
const Rectangulo = require('./Rectangulo');

const Rectangulo1 = new Rectangulo("Naranja", 0, 1, 5);
const Circulo1 = new Circulo("Amarillo", 0, 3);

console.log(Rectangulo1.mostrarInfo());
console.log(Circulo1.mostrarInfo());

File: 10ejercicioscompletos/6.Formas_Geometricas/Control.py
from Circulo import Circulo
from Rectangulo import Rectangulo

def control():
    forma1 = Circulo("Naranja", 9)
    forma2 = Rectangulo("Verde", 16, 3)

    formas = [forma1, forma2]

    for f in formas: 
        f.mostrar_informacion()

control()